pkgs/applications/science/math/geogebra/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, jre, makeDesktopItem, makeWrapper, unzip, language ? "en_US" }:
 let
   pname = "geogebra";
-  version = "5-0-745-0";
+  version = "5-0-783-0";
```

Comment:
5-0-785-0 is available.

Reply:
Thanks, I updated the pull request accordingly.